ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -406,6 +405,16 @@ pub(crate) enum Instruction {
         binop: BinOp,
         rhs: Operand,
     },
+    #[deku(id = "12")]
+    DeoptSafepoint {
+        id: Operand,
+        // FIXME: this is never used, do we need to serialize this?
+        num_shadow_bytes: Operand,
```

Comment:
No, we won't need this. We can get rid of it. But given we will merge deopt into the call/br instructions they are attached to soon, there's probably no need to do this now.